test/unit/jobs/cleanup-test.js

```diff
@@ -103,6 +110,39 @@ describe("Delete unused files", () => {
     }, 200);
   });
 
+  it("should delete financial data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
+      cleanupJob.financial.findByKey("risefinancial_realtime_HXXHCTR5AQFQ_-Kn5MaUE2cpYN-FCQSOU_1M_", (err, foundFinancial) => {
+        expect(Object.keys(foundFinancial.data).length).to.equal(0);
+
+        done();
+      })
+    }, 200);
+  });
+
+  it("should delete spreadsheet data that has not been accessed within the last 24 hours", (done) => {
+    // Wait for file to be deleted before running test.
+    setTimeout(function() {
```

Comment:
Instead of using a timeout, can the cleanup job constructor function just construct the object rather than setting off the side effect of running the deletion?  Then cleanupDatabases() can be run separately in the test, and it can return a promise (or maintain the callback style).